rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"

```

Comment:
Right. Darn it, that probably wouldn't've been a problem had we settled on a moniker! :ghost: